controllers/validators.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  validateId(req, res, next, id) {
+    const productId = Number(id);
```

Comment:
Since we are sharing this validator, maybe rename it to something generic like resourceId